### DISCUSSION TITLE
New-HTMLStatusItem

### DISCUSSION THREAD

**Discussion by boncz92** [EvotecIT/PSWriteHTML#321] [Q&A] [answered]
It states that this parameter is going to be depreciated, is there a new item I should be using instead of this one? I don't know of another function that offers the same functionality.

**Comment by PrzemyslawKlys** [comment, marked as answer]
Hi, `New-HTMLStatusItem` is not deprecated. Just some parameters are. 

This is because the old, very limited way. You could only choose from 3 icons (good, bad, dead) and from 5 percentages (0,10,30,70,100). This would give you limited icon and 5 colors. 

At some point in time we have added all FontAwesomeIcons and the ability to use 800 colors, 800 background colors and ability to use hex if that is not enough. 

In other words from 5 different parameter sets only 1st one is deprecated.

```powershell
NAME
    New-HTMLStatusItem
    
SYNTAX
    New-HTMLStatusItem [-Name <string>] [-Status <string>] [-Icon {Dead | Bad | Good}] [-Percentage {0% | 10% | 30% | 70% | 100%}] [-FontColor <string>] [<CommonParameters>]
    
    New-HTMLStatusItem [-Name <string>] [-Status <string>] [-FontColor <string>] [-BackgroundColor <string>] [-IconHex <string>] [<CommonParameters>]
    
    New-HTMLStatusItem [-Name <string>] [-Status <string>] [-FontColor <string>] [-BackgroundColor <string>] [-IconSolid <string>] [<CommonParameters>]

    New-HTMLStatusItem [-Name <string>] [-Status <string>] [-FontColor <string>] [-BackgroundColor <string>] [-IconRegular <string>] [<CommonParameters>]

    New-HTMLStatusItem [-Name <string>] [-Status <string>] [-FontColor <string>] [-BackgroundColor <string>] [-IconBrands <string>] [<CommonParameters>]
```

**Reply by boncz92** [reply]
Ahhh I understand now, the warning was throwing me off. I appreciate the help. One other question, where do I find the -IconHex options? I was looking around and cant find a definitive answer on the github.

**Reply by PrzemyslawKlys** [reply]
Here's the PR that enabled it:
- https://github.com/EvotecIT/PSWriteHTML/pull/111

> The $IconHex works with UTF-8 Unicode character hex codes. All the main browsers support translation of these codes by default and a reference sheet of codes can be found at https://www.w3schools.com/charsets/ref_html_utf8.asp.

**Reply by PrzemyslawKlys** [reply]
FontAwesome is easier to use as it should give you choice from around 1000 icons
